packages/mds-agency-cache/index.ts

```diff
@@ -417,7 +408,7 @@ async function readDevicesStatus(query: {
       try {
         const parsedItem = parseEvent(item)
         if (
-          EVENT_STATUS_MAP[parsedItem.event_type] === VEHICLE_STATUSES.removed ||
+          EVENT_STATUS_MAP[parsedItem.event_type] === 'removed' ||
```

Comment:
Just curious, why? It seems safer to use the enum in case there's a typo, in general.